morse_module/morse.c

```diff
@@ -197,7 +197,9 @@ int main() {
 		
 	state = READY_2_START;
 
-	while(state == READY_2_START);
+	while(state == READY_2_START) {
+		sleep_ms(10);
```

Comment:
if you want it to wait the minimum amount, 0 is better, but whatever you feel like